_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1131,6 +1132,7 @@ mfxStatus VAAPIEncoder::CreateAccelerationService(MfxVideoParam const & par)
         return MFX_ERR_DEVICE_FAILED;
 
     attrib[1].value = vaRCType;
+    attrib[2].value = m_core->GetSession()->m_priority;
```

Comment:
Why we don't make mapping here?

Reply:
Pending on the VAAPI definition. When VAAPI definition is finalized, it will be updated.

Reply:
updated.